.evergreen/run-tests.sh

```diff
@@ -201,6 +201,12 @@ if [ -n "$COVERAGE" -a $PYTHON_IMPL = "CPython" ]; then
     fi
 fi
 
+if $PYTHON -c 'import dns'; then
+  # Trying with/without --user to avoid:
+  # ERROR: Can not perform a '--user' install. User site-packages are not visible in this virtualenv.
+  $PYTHON -m pip install --upgrade --user 'dnspython<2.0.0' || $PYTHON -m pip install --upgrade 'dnspython<2.0.0'
```

Comment:
Why not always install without `--user`?

Reply:
Because then we might need `sudo`. This will just be a temporary workaround until the toolchain is updated.